tests/scripts/get_list_for_release_note.sh

```
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -e
+
+# NOTE: We can skip authenticating github username and token but that will limit the github api request to 60request/IP/hr and by authenticating the limit will be 5000.
+USERNAME=""
+TOKEN=""
+
+# This is just example branch name, change according to uses
+FROM_BRANCH="upstream/release-1.6" # example branch
+TO_BRANCH="v1.6.7" # example branch
+
+
+# for minor releases
+function minor_release() {
+for pr in $(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_BRANCH} | grep pull | sed -e "s/.*#\([0-9]\+\).*/\1/g" | sort -rn | uniq); do
+	backport_prs=$(curl -u ${USERNAME}:${TOKEN} https://api.github.com/repos/rook/rook/pulls/${pr} 2>/dev/null | jq '.title';)
```

why ignoring errors?

---

which errors?

---

@leseb I see, removed `2>/dev/null` added `-s`